src/pages/strategy/tabs/playground/README.md

```diff
@@ -0,0 +1,12 @@
+This Strategy Room Tab is used as a sandbox for helping you with KC3 development.
+
+Not that all changes to this directory should not be commited to the repository,
+you can do the following so Git will ignore all changes you made to this directory:
+
+```bash
+$ cd </path/to/repo/home>
+$ cd src/pages/strategy/tabs/playground
+$ git update-index --assume-unchanged *
```

Comment:
nice I use this too in other repos.

additional, if one is using tortoisegit, they can right-click the file in the commit dialog, then choose "assume unchanged"